examples/Atlas/+atlasControllers/QTrajEvalBlock.m

```diff
@@ -72,7 +72,7 @@
       obj.use_error_integrator = options.use_error_integrator;
       [obj.jlmin,obj.jlmax] = getJointLimits(obj.robot);
       obj.nq = getNumPositions(r);
-      obj.eta = 0.995;  % decay integral over time
+      obj.eta = 1.0;  % decay integral over time
```

Comment:
This is apparently from a commit from @kuindersma from February 13.
https://github.com/tkoolen/drake/commit/7c75e5c7ad666b69b2313c9a91fe7a70ccfd2d0d

Did this get included because of merging in mitdrc/drake master at some point?